cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -206,47 +263,49 @@ <h2 class="h3 u-mb15">
+                    {% for purchase_apr in purchase_aprs %}
                         <tr>
                             <td>{{ credit_tiers[loop.index][1] }}</td>
                             <td>{{ apr(purchase_apr) }}</td>
-                            {# TODO: Put in actual ratings #}
                             <td>
-                                <span class="a-credit-card-apr-rating--average">
-                                    {{ svg_icon("dollar-round") }}
-                                    {{ svg_icon("dollar-round") }}
-                                    Average for this range
-                                </span>
+                                {% if purchase_apr_ratings[loop.index0] is not none %}
+                                    {% set label = purchase_apr_rating_labels[purchase_apr_ratings[loop.index0]] %}
+                                    <span class="a-credit-card-apr-rating--{{ label }}">
+                                        {%- for i in range(purchase_apr_ratings[loop.index0] + 1) %}
+                                            {{ svg_icon("dollar-round") }}
+                                        {% endfor -%}
+
+                                        Pay {{ label }} interest
+                                    </span>
+                                {% elif purchase_aprs == 'no data' %}
+                                    N/A
+                                {% else %}
+                                    Not targeted for these credit scores
+                                {% endif %}
                             </td>
                         </tr>
                     {% endfor %}
                 </tbody>
             </table>
-        {% elif card.purchase_apr_median is not none
-            and card.purchase_apr_min != card.purchase_apr_max %}
             <p>
-                You’ll only know your exact APR if you are approved for the card.
-                The median APR for this card is {{ apr(card.purchase_apr_median) }}.
+                The purchase APR is
+                {% if 'V' in card.index %}
+                    variable, meaning it can go up or down based on an index. You can
+                    <a href="/ask-cfpb/what-is-the-difference-between-a-fixed-apr-and-a-variable-apr-en-45/">learn
+                    more about variable APRs in general</a> or expand this section to
+                    learn more about the index this APR is based on.
+                {% else %}
+                    fixed, meaning it will not go up or down with changes to an index
+                    but may still change. You can
+                    <a href="/ask-cfpb/what-is-the-difference-between-a-fixed-apr-and-a-variable-apr-en-45/">learn
+                    more about fixed APRs.</a>
+                {% endif %}
             </p>
+        {% elif card.purchase_apr_offered and purchase_aprs == 'no data' %}
+            <p>The issuer did not provide APR information for this card.</p>
```

Comment:
I wish we could bold this and put a frowny face to highlight problematic oversights like this.